src/pages/Touringcar/components/MapPanel/StopDetails.tsx

```diff
@@ -49,94 +48,70 @@ const ParkingSpaceDetailFeature = () => {
     enabled: true,
     queryKey: ['panoramaThumbnail'],
     queryFn: () => {
-      if (!currentParkingSpace?.geometry.coordinates.length) {
+      if (!currentStop?.geometry.coordinates.length) {
         throw new Error('Coordinates are required.')
       }
 
       return getPanoramaThumbnail({
-        lat: currentParkingSpace?.geometry.coordinates[1],
-        lon: currentParkingSpace?.geometry.coordinates[0],
+        lat: currentStop?.geometry.coordinates[1],
+        lon: currentStop?.geometry.coordinates[0],
```

Comment:
Sanity check: Are both coordinates required? Than we should not only test if they have a length but that the length is 2 (or larger/equal than 2)?

Reply:
Good point. I'll test for a length of 2. It should always be 2 because it is a geojson Point (tuple).